### DISCUSSION TITLE
Adding constraints only for some variables

### DISCUSSION THREAD

**Discussion by aabsz** [hilo-mpc/hilo-mpc#38] [General]
Hello,

I'm interested in applying constraints to specific input variables. Is it possible to do so, or is it only feasible to define upper and lower bounds for all variables universally?

**Comment by brunomorampc** [comment]
yes you can do it.  Just past a list [lb1, lb2, ..., lbn] fo the bounds. 
Can me an example code of what you want to do? So I can be more precise with my answer

**Reply by aabsz** [reply]
Thank you. The problem has been solved now
